examples/ruby/run

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+source ~/.rvm/scripts/rvm
+RUBY_VERSION="2.7.2"
+
+function announce() {
+  echo "++++++++++"
+  echo "$@"
+  echo "++++++++++"
+}
+
+if [[ ! -d "./out/ruby" ]]; then
+  announce "Client not found. Generating..."
+  ./bin/generate_client ruby > /dev/null
+  echo
+fi
+
+if [[ -z "$(docker-compose ps -q)" ]]; then
+  announce "Environment not found. Spinning up..."
+  ./bin/start_conjur > /dev/null
+  echo
+fi
+
+if [[ ! -f ./out/ruby/openapi_client-1.0.0.gem ]]; then
+  announce "Building OpenAPI gem..."
+  gem build ./out/ruby/openapi_client.gemspec -C ./out/ruby/
+  echo
+fi
+
+announce "Creating new RVM Gemset and installing OpenAPI gem..."
+if [[ ! $(rvm list | grep "ruby-$RUBY_VERSION") ]]; then
+  rvm install $RUBY_VERSION
+fi
+rvm use $RUBY_VERSION
```

Comment:
Since RVM is not a common dev tool, some sort of check for its availability at the beginning might need to be done.